editor/project_manager/project_dialog.cpp

```diff
@@ -315,6 +315,10 @@ void ProjectDialog::_create_dir_toggled(bool p_pressed) {
 			target_path = target_path.path_join(last_custom_target_dir);
 		}
 	} else {
+		// Strip any trailing slash.
+		while (target_path.ends_with("/") || target_path.ends_with("\\")) {
+			target_path = target_path.substr(0, target_path.length() - 1);
+		}
```

Comment:
```suggestion
		target_path = target_path.rstrip("/");
```

Reply:
Ah, much nicer! Any reason not to include backslash?
```suggestion
		target_path = target_path.rstrip("/\\");
```

Reply:
`rstrip` replaces all occurrences of a pattern from the right side, so it would only replace it if it ends with `/\` with the proposed change.
If you want to strip both, you have to do them separately: `target_path.rstrip("/").rstrip("\\")`

Godot shouldn't be using backslashes in paths though, but I guess users might input them manually / copy from the Windows explorer, so that makes sense.